x-pack/plugins/security_solution/public/management/components/effected_policy_select/effected_policy_select.test.tsx

```diff
@@ -161,5 +165,44 @@ describe('when using EffectedPolicySelect component', () => {
       selectPerPolicy();
       expect(queryByTestId('loading-spinner')).not.toBeNull();
     });
+
+    it('should hide policy link when no policy management privileges', () => {
+      (useUserPrivileges as jest.Mock).mockReturnValue({
+        ...initialUserPrivilegesState(),
+        endpointPrivileges: {
+          loading: false,
+          canWritePolicyManagement: false,
+          canReadPolicyManagement: false,
+        },
+      });
+      const { queryByTestId } = render({ isGlobal: false });
+      expect(queryByTestId('test-policyLink')).toBeNull();
+    });
+
+    it('should show policy link when all policy management privileges', () => {
+      (useUserPrivileges as jest.Mock).mockReturnValue({
+        ...initialUserPrivilegesState(),
+        endpointPrivileges: {
+          loading: false,
+          canWritePolicyManagement: true,
+          canReadPolicyManagement: true,
```

Comment:
nit: I believe these are already true on the initial state.

Reply:
Exactly, I'm setting it manually to `true` just to ensure it will keep working on future changes, so we don't rely on the initial state, let me know if that make sense 🙂

Reply:
Absolutely 👍 Also it's easier to reason when reading the test.